sdk/keyvault/azure-security-keyvault-certificates/vcpkg/Config.cmake.in

```diff
@@ -4,7 +4,7 @@
 @PACKAGE_INIT@
 
 include(CMakeFindDependencyMacro)
-find_dependency(azure-core-cpp "1.2.0")
+find_dependency(azure-core-cpp "1.3.1")
```

Comment:
I don't think this change alone is enough.

I think you'll also need to modify the json file:
https://github.com/Azure/azure-sdk-for-cpp/blob/9b4672be9324f218ecdedfe85cae184a59e2e482/sdk/keyvault/azure-security-keyvault-secrets/vcpkg/vcpkg.json#L17

cc @antkmsft, @ras0219-msft